.ruby-version

```diff
@@ -0,0 +1 @@
+2.2.4
```

Comment:
Was there a specific reason why you commited `.ruby-version`? To my knowledge this is not a best practise. Please remove this commit.